### ISSUE TITLE
Can't get url from InAppBrowser event

### ISSUE THREAD

**Issue by sydneyhenrard** [driftyco/ng-cordova#565]
I try to get the url from the loadstop event but the event does not have a `url` property. A `console.log` on the event displays:

```
Object
currentScope: null
defaultPrevented: false
name: "loadstop"
preventDefault: function () {
stopPropagation: function () {stopPropagation = true;}
targetScope: ChildScope
__proto__: Object
```

My code in the controller:

```
var inApp = $cordovaInAppBrowser.init({
	location: 'no',
	clearcache: 'yes'
});

inApp.$on('loadstop', function(event) {
	console.log('loadstop');
	console.log(event);
	var url = event.url;
	console.log(url);
	var filename = url.substring(url.lastIndexOf('/')+1);
	console.log(filename);
	if(filename == "mobile.login.html"){
		//alert("Success");
		$cordovaInAppBrowser.close();
		//$state.go('logged-in');
	}
});

$scope.googleLogin = function() {
	$cordovaInAppBrowser
		.open('http://192.168.0.27:3000/auth/google', '_blank')
		.then(function(event) {
			// success
		})
		.catch(function(event) {
			// error
		});
}
```

Ionic application ran in Genymotion emulator (Android 4.4.4)

**Comment by pbernasconi** [comment]
There have been changes made to the InAppBrowser in the `master` branch. What version of ngCordova are you using?

**Comment by sydneyhenrard** [comment]
ngCordova v0.1.9-alpha

**Comment by sydneyhenrard** [comment]
If I use cordova plugin it works:

```
var x = $window.open('http://192.168.0.27:3000/auth/google', '_blank');
x.addEventListener('loadstop', function (event) {
});
```

**Comment by pbernasconi** [comment]
I made some changes yesterday that I have tested and are working. Install the `master` branch by:

```
bower install ngCordova#master
```

#### Using is like so:
**Set Default Options**
```
.config(function($cordovaInAppBrowserProvider) {
  var options = {
    location : "no",
    toolbar: "yes"
  };

  $cordovaInAppBrowserProvider.setDefaultOptions(options);
});
```

**Open the browser in a controller**
```
.controller("SomeCtrl", function ($cordovaInAppBrowser) {

  var overrideOptions = {
    location : "yes",
    toolbar: "yes"
  };

  $cordovaInAppBrowser.open('http://ngcordova.com', '_blank', options).then(function () {
      console.log("InAppBrowser opened http://ngcordova.com successfully");
   }, function (error) {
      console.log("Error: " + error);
   });

  // listen for inappbrowser events
  $rootScope.$on('$cordovaInAppBrowser:loadstart', function(e, event) {});
  $rootScope.$on('$cordovaInAppBrowser:loadstop', function(e, event));
  $rootScope.$on('$cordovaInAppBrowser:loaderror', function(e, event));
  $rootScope.$on('$cordovaInAppBrowser:exit', function(e, event));
});
```

**Comment by sydneyhenrard** [comment]
It's working, I can now get the event. Can you check that the default option are correctly applied. If I configure with no and no, I still see the location bar. If I override the options when the `open` is called, I don't see the location bar as expected.

```		
var options = {
	location: "no",
	toolbar: "no"
};
$cordovaInAppBrowserProvider.setDefaultOptions(options);
}
```

Thanks for the quick fix

**Comment by pbernasconi** [comment]
You're right, I made a small mistake fixing now.

**Comment by pbernasconi** [comment]
Fix now in the `master` branch: 17668fdd2efcfb624b5bf2fbdb46658e44765085
